packages/testing-helpers/src/lib.js

```diff
@@ -1,10 +1,8 @@
-import { TemplateResult } from 'lit-html';
-
 export const isIterable = object => object != null && typeof object[Symbol.iterator] === 'function';
 
 function isValidNonIterableRenderArg(x) {
   return (
-    x instanceof TemplateResult ||
+    typeof x._$litType$ !== 'undefined' ||
```

Comment:
Iffy. Happy to hear better ideas about this one.

Reply:
This is ok for now. We'll have to land on a final solution and document it on our end.

Reply:
We now have `isTemplateResult()` in module `lit-html/directive-helpers.js`.

https://github.com/Polymer/lit-html/blob/lit-next/packages/lit-html/CHANGELOG.md#added